src/kirby/components/card/card.component.scss

```diff
@@ -22,6 +22,10 @@ $min-card-width: 304px;
   min-width: $min-card-width;
   overflow: hidden;
 
+  &.padding {
```

Comment:
Right now the whole card gets default padding, which is also visible on the card-example badges. I think we maybe should consider creating a card-content component and apply the right default padding to the card-title, card-content and card-footer. In this way we will have a more flexible solution.

Reply:
I've just talked with UX and this is how they want it for now. We will therefore keep the padding on the whole card.